src/gentropy/dataset/dataset.py

```diff
@@ -153,6 +153,33 @@ def from_parquet(
             raise ValueError(f"Parquet file is empty: {path}")
         return cls(_df=df, _schema=schema, **class_params)
 
+    @classmethod
+    def from_json(
```

Comment:
I'd make `from_parquet` capable of taking diverse formats as input, but maybe for another PR

Reply:
`from_parquet("gs:/some_file.json")` is semantically wrong. We could make the method more dynamic though, as the implementation is the same between `from_parquet` and `from_json`. Either way it is no longer needed for the gold standard, I will remove it.